### PR TITLE
[IMP] CRM: merge leads improvements

### PR DESCRIPTION
**Pull request by jero-odoo**
Updating the CRM: Merge similar leads doc to correct a typo, as well as update icons.

doc task: ﻿[﻿https://www.odoo.com/mail/view?model=project.task&res_id=4014674&access_token=aa2e42e2-7139-4e31-9a54-729c46e7a56e](https://www.odoo.com/mail/view?model=project.task&res_id=4014674&access_token=aa2e42e2-7139-4e31-9a54-729c46e7a56e)

No significant content changes, sending directly to tech review

### CODE REVIEW

**Comment by jero-odoo** [comment]
@samueljlieber Since this did not have any significant content changes, I am sending this to tech review. Thanks!

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 1]
```diff
@@ -40,14 +41,15 @@ Merging similar leads and opportunities
...
 After confirming that the leads/opportunities should be merged, return to the Kanban view using the
-breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the :guilabel:`≣
-(view list)` icon to change to list view.
+breadcrumb link, or by clicking the :guilabel:`Similar Leads` smart button. Click the
+:icon:`oi-view-list` :guilabel:`(list)` icon to change to list view.
 
 Check the box on the left of the page for the leads/opportunities to be merged. Then, click the
-:guilabel:`⚙️ Actions` icon at the top of the page, to reveal a drop-down menu. From that drop-down
-menu, select the :guilabel:`Merge` option to merge the selected opportunities or leads.
+:icon:`fa-cog` :guilabel:`(Actions)` icon at the top of the page, to reveal a drop-down menu. From
```
Comment (samueljlieber) [inline content/applications/sales/crm/pipeline/merge_similar.rst:48]: In this instance, "Actions" text is visible in the button as well :)

Screenshot 2024-06-28 at 12 27 45 PM

```suggestion
:icon:`fa-cog` :guilabel:`Actions` icon at the top of the page, to reveal a drop-down menu. From
```

File: content/applications/sales/crm/pipeline/merge_similar.rst [hunk 2]
```diff
@@ -40,14 +41,15 @@ Merging similar leads and opportunities
...
+that drop-down menu, select the :guilabel:`Merge` option to merge the selected opportunities or
+leads.
 
-When :guilabel:`Merge` is selected from the :guilabel:`⚙️ Actions` drop-down menu, a
+When :guilabel:`Merge` is selected from the :icon:`fa-cog` :guilabel:`(Actions)` drop-down menu, a
```
Comment (samueljlieber) [inline content/applications/sales/crm/pipeline/merge_similar.rst:52]:
```suggestion
When :guilabel:`Merge` is selected from the :icon:`fa-cog` :guilabel:`Actions` drop-down menu, a
```

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo, thanks for this quick fix PR. Your changes look good, just a couple small adjustments needed. Approving 👍
..
@robodoo delegate=jero-odoo

**Comment by jero-odoo** [comment]
@robodoo r+

**Pull request closed without merging**